clarified healing rules and changed to 3 actions

bleehu opened the pull request:
modified:   01_01_Basic_Rules.txt
	modified:   Combat Rules/1_Combat_Rules.txt
	modified:   Items/07_Items.txt

should address issue #673

bleehu committed:
clarified healing rules and changed to 3 actions
	modified:   01_01_Basic_Rules.txt
	modified:   Combat Rules/1_Combat_Rules.txt
	modified:   Items/07_Items.txt

bleehu commented:
I think it's an important balance change to make healing a wound take longer than aiming and firing a gun. It radically changes our action economy. With this significant nerf to med kits and repair kits, we may consider dropping the price for them.

Turtlelord26 commented on 01_01_Basic_Rules.txt:
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
+adventure that you need to heal a wound or even stabalize a friend who is on the
+doorstep of death. The first thing you'll need to know is that treating the 
+injuries of a cybernetic character usually requires the repair skill, while 
+treating a biological character requires the medicine skill. Items such as a 
```
Gr: An item such as a ...

Turtlelord26 commented on 01_01_Basic_Rules.txt:
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
+injuries of a cybernetic character usually requires the repair skill, while 
+treating a biological character requires the medicine skill. Items such as a 
+medkit or repair kit is required for most characters to be able to treat a wound
+or stabalize a dying character. For stabalizing a dying character, see the 
```
sp: Stabilize. Several incidences.

Turtlelord26 commented on 01_01_Basic_Rules.txt:
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
+or stabalize a dying character. For stabalizing a dying character, see the 
+"Stabalizing" section under "Death and Dying" in the Combat Rules.
+
+    Using a medkit or a repair kit to heal a wounded character takes 3 actions
```
Order suggestion: Using a ... to heal a wounded character in combat takes 3 Actions.

Turtlelord26 commented on 01_01_Basic_Rules.txt:
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
+    Using a medkit or a repair kit to heal a wounded character takes 3 actions
+in combat. Each healing application heals only one wound at a time. In order for
+any healing to happen, the healer must make a repair or medicine skill check 
+where the DC is the ammount of damage that the wound inflicted. The ammount of 
```
Sp: Amount. 2 incidences.

Turtlelord26 commented on 01_01_Basic_Rules.txt:
```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
+	For example, Buster gets shot by an SMG for 40 damage and gets shot again by
+a sniper for 80 damage. Buster now has two wounds, the SMG hit and the sniper 
+hit. Buster's friend Sheela uses a medkit to heal the SMG hit. Sheela has a med
+skill of 10, a dex mod of 8 and rolls a 45. 
```
Clarity point: does the targeted wound have to be declared before the roll, or can it be after? This should be stated explicitly.

Turtlelord26 commented on Combat Rules/1_Combat_Rules.txt:
```diff
@@ -239,15 +239,18 @@ above. Even if you had 5 health and took 80 damage, when you go down you go to 0
+minimum two. Past this limit, stabilized characters are Unconscious in addition 
+to the other effects of Stabilization. Note that most persistent damage effects, 
+like bleeds or burns, fall off of characters when they are Downed. If the 
+stabalizing character starts stabalizing with their last action or two, 
```
Sp: Stabilize

Turtlelord26 reviewed (changes requested):
I am not opposed to testing out slow-heal. I have been thinking about nerfing the amount of each heal, but slowing them down has much the same effect and is certainly worth investigating.
EDIT: Also, the flavor of your way is better. 🐢 

Before greenlighting, I will ask you to additionally apply this nerf to relevant Doc and Engi abilities. I see class balance suffering a lot otherwise.

rosebrallier commented:
> Before greenlighting, I will ask you to additionally apply this nerf to relevant Doc and Engi abilities. I see class balance suffering a lot otherwise.
Perhaps the Doc and Eng abilities should take 2 actions? 3 would be incredibly punishing to their action economy, and would encompass the idea that they heal faster because they've had lots of practice.

bleehu committed:
spelling and nerfed engineer and doctor proceedures to take longer than gunshots
	modified:   ../01_01_Basic_Rules.txt
	modified:   ../CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt
	modified:   ../CharacterCreation/Class-Specific Documentation/Engineer Processes.txt
